go.mod

```diff
@@ -8,7 +8,7 @@ go 1.18
 require (
 	github.com/chzyer/readline v1.5.0
 	github.com/google/uuid v1.3.0
-	github.com/multiprocessio/datastation/runner v0.0.0-20220616090857-d42388371862
+	github.com/multiprocessio/datastation/runner v0.0.0-20220628000637-8472e7fd559e
```

Comment:
Might as well bump this to [5506ab9](https://github.com/multiprocessio/datastation/commit/5506ab9ea69864ffd0b037a2fa24bed0970f7c21) to take in your parquet fix!